content/20-exploring-the-cluster/22-rancher-applications/index.en.md

```diff
@@ -1,43 +1,42 @@
 ---
-title: "Rancher Applications"
+title: 'Rancher Applications'
 weight: 22
 ---
 
 In the following module, you will complete the outcomes listed below:
 
 1. Use the Rancher App Catalog to deploy Longhorn and NeuVector
-2. Explore features inside of Longhorn and NeuVector.
+2. Explore features of Longhorn and NeuVector.
 
 ---
 
-
 # Deploying the Rancher Applications
 
-In Exploring the Cluster, we briefly looked at the Rancher App Catalog, but now we are going dive deeper into it with two of the Rancher Kubernetes applications (Longhorn and NeuVector).
+In Exploring the Cluster, we briefly looked at the Rancher App Catalog, but now we are going dive deeper by installing two Rancher Kubernetes applications: Longhorn and NeuVector.
```

Comment:
Change to "In Exploring the Cluster, we briefly looked at the Rancher App Catalog, but now we are going dive deeper by installing two core Rancher Kubernetes applications: Longhorn and NeuVector."?